[2024-09-19T21:03:58.051+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-19T21:03:58.071+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline.run_data_processing manual__2024-09-19T21:02:32.699115+00:00 [queued]>
[2024-09-19T21:03:58.083+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline.run_data_processing manual__2024-09-19T21:02:32.699115+00:00 [queued]>
[2024-09-19T21:03:58.083+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2024-09-19T21:03:58.099+0000] {taskinstance.py:2888} INFO - Executing <Task(DockerOperator): run_data_processing> on 2024-09-19 21:02:32.699115+00:00
[2024-09-19T21:03:58.107+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'nba_data_pipeline', 'run_data_processing', 'manual__2024-09-19T21:02:32.699115+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline.py', '--cfg-path', '/tmp/tmpn8o2dnrh']
[2024-09-19T21:03:58.110+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask run_data_processing
[2024-09-19T21:03:58.112+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=296) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-19T21:03:58.114+0000] {standard_task_runner.py:72} INFO - Started process 297 to run task
[2024-09-19T21:03:58.164+0000] {task_command.py:467} INFO - Running <TaskInstance: nba_data_pipeline.run_data_processing manual__2024-09-19T21:02:32.699115+00:00 [running]> on host 19c433e2f996
[2024-09-19T21:03:58.264+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline' AIRFLOW_CTX_TASK_ID='run_data_processing' AIRFLOW_CTX_EXECUTION_DATE='2024-09-19T21:02:32.699115+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-19T21:02:32.699115+00:00'
[2024-09-19T21:03:58.266+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-19T21:03:58.298+0000] {docker.py:367} INFO - Starting docker container from image us-west1-docker.pkg.dev/nba-fantasy-ml/nba-data-pipeline-repo/data-processing:latest
[2024-09-19T21:03:58.308+0000] {docker.py:375} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-09-19T21:03:59.117+0000] {docker.py:438} INFO - 2024-09-19 21:03:59,116 - INFO - Starting data processing...
[2024-09-19T21:03:59.122+0000] {docker.py:438} INFO - 2024-09-19 21:03:59,120 - INFO - Database connected.
[2024-09-19T21:03:59.122+0000] {docker.py:438} INFO - 2024-09-19 21:03:59,120 - INFO - Cleaning and preprocessing data...
[2024-09-19T21:03:59.160+0000] {docker.py:438} INFO - 2024-09-19 21:03:59,158 - INFO - Validation passed.
[2024-09-19T21:03:59.160+0000] {docker.py:438} INFO - 2024-09-19 21:03:59,158 - INFO - Assigning unique IDs for players and games...
[2024-09-19T21:03:59.242+0000] {docker.py:438} INFO - 2024-09-19 21:03:59,241 - INFO - Inserting player stats into database...
[2024-09-19T21:03:59.344+0000] {docker.py:438} INFO - 2024-09-19 21:03:59,340 - INFO - Data processing completed successfully.
[2024-09-19T21:03:59.345+0000] {docker.py:438} INFO - 2024-09-19 21:03:59,341 - INFO - Database connection closed.
[2024-09-19T21:03:59.345+0000] {docker.py:438} INFO - 2024-09-19 21:03:59,341 - INFO - Successfully processed player_box_scores/2023-24/nba_player_stats_2023-24_2023-10-24_to_2023-10-25_2024-09-19_21-03-56.csv
[2024-09-19T21:03:59.541+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-19T21:03:59.541+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=nba_data_pipeline, task_id=run_data_processing, run_id=manual__2024-09-19T21:02:32.699115+00:00, execution_date=20240919T210232, start_date=20240919T210358, end_date=20240919T210359
[2024-09-19T21:03:59.567+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-19T21:03:59.583+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-19T21:03:59.585+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
